cli/azd/pkg/alpha/alpha_feature_manager.go

```diff
@@ -77,32 +79,32 @@ func (m *FeatureManager) IsEnabled(featureId FeatureId) bool {
 	// guard from using the alphaFeatureManager from multiple routines. Only the first one will create the cache.
 	m.withSync.Do(m.initConfigCache)
 
+	enabled := false
+
 	// For testing, and in CI, allow enabling alpha features via the environment.
-	envName := fmt.Sprintf("AZD_ALPHA_ENABLE_%s", strings.ToUpper(string(featureId)))
+	envName := fmt.Sprintf("AZD_ALPHA_ENABLE_%s", strings.ReplaceAll(strings.ToUpper(string(featureId)), ".", "_"))
```

Comment:
Is this a breaking change? 
If we want to support `_`, can we do it w/o removing the `.` ?

Reply:
I feel like it is a better experience when using environment variable override.  Example: Instead of `AZD_ALPHA_ENABLE_AKS.HELM` it is `AZD_ALPHA_ENABLE_AKS_HELM`

Reply:
agree. Can we support both so we don't break anyone currently relaying on the `FOO.BAR` ?

Reply:
`.` isn't a valid variable ID character in many shells, and I don't see any existing [code references](https://github.com/search?q=AZD_ALPHA_ENABLE&type=code) in public repos.

```bash
bash-3.2$ export ASD.WHAT='true'
bash: export: `ASD.WHAT=true': not a valid identifier
```

I'd prefer if we break it, the uptake isn't too bad.